### PR TITLE
Mosaic-Modal

### PR DESCRIPTION
**Pull request by jase-d**
Modal is starting to take shape:

1: Initially created the modal using react-modal
2: figured out styling template and was able to modify its size
3: added a css animation and transition to simulate the UI of airbnb
Check out the GIF: [HERE](https://recordit.co/Nao92M0pvJ)

### CODE REVIEW

File: client/src/components/mosaic_small.jsx [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
         <div  className="frames"
               id="pic-frame">
-                
-              {photoFrame.map((url, index) => <div style={{ backgroundImage: `url(${url})`} }
+
+              {photoFrame.map((url, index) =>
+              <div style={{ backgroundImage: `url(${url})`} }
                                                    id={ 'ms-' + index }
-                                                   class="boxes" ></div>)}
+                                                   class="boxes"
```
Comment (bland0) [inline client/src/components/mosaic_small.jsx:22]: Are you sure this shouldn't be className?

**Review by bland0 — CHANGES_REQUESTED** [review]
Not sure you meant to commit bundle.js

File: client/src/components/mosaic_small.jsx (hunk 1 above)
Comment (javsflores) [reply client/src/components/mosaic_small.jsx:22]: interesting, I wonder if this works, I thought for react it needed to be "className"

File: package.json [hunk 2]
```diff
@@ -38,6 +38,7 @@
     "path": "^0.12.7",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-modal": "^3.11.2",
```
Comment (javsflores) [inline package.json:41]: remind me to ask you about this dependencies later, I'm curious to know what it does. Good job!

**Comment by jase-d** [comment]
> Not sure you meant to commit bundle.js

Honestly I've been trying to figure that out for a bit now, I've added the line to ignore but it won't grey out. I've even removed the bundle file, commited, and ran it again but won't ignore.

**Pull request merged**